powersimdata/network/constants/plants.py

```diff
@@ -60,6 +60,16 @@
     "ng": 469,
 }
 
+# MWh_electric to MWh_thermal
```

Comment:
Should it be MWh_termal to MWh_electric? The supply of x MW of heat from burning coal/dfo/ng (input) for every y MW of electricity (output) it produces

Reply:
The ratio `efficiency` given below is `MWh_electric`/`MWh_thermal`. So the electric power output a generator produces for burning 1 MWh of thermal energy. Does that make sense?